Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -4,38 +4,53 @@ import Nimble
 import XCTest
 
 final class ChangeStreamTest: MongoSwiftTestCase {
```

Comment:
I think a test verifying the `fullDocument` `Codable` integration works + `CollectionType` interop would be nice to have.

Reply:
Looks good. I think one more test verifying that the `CollectionType` is used by default (when watching collections) would also be useful.